timesketch/frontend-ng/src/components/Explore/EventList.vue

```diff
@@ -839,10 +856,15 @@ export default {
           fileLink.href = fileURL
           fileLink.setAttribute('download', fileName)
```

Comment:
Comment on the `let fileName = 'export.zip'`, what do you think about adding timesketch in there? Like `timesketch_export.zip`? Best would also be to append a timestamp as well to prevent the pile of `export (1-100).zip` files in the Download folder.

This would allow for faster identification what those export archives in the download folder actually are.

Reply:
Sure, this was ported over from the old code and we haven't changed that. I'll rename it.

Reply:
The new format is not timesketch_export_20230401T12.....csv